src/woo_publications/publications/models.py

```diff
@@ -191,6 +192,41 @@ class Publication(ModelOwnerMixin, models.Model):
         ),
     )
 
+    # Retention fields:
+    bron_bewaartermijn = models.CharField(
+        _("retention policy source"),
+        help_text=_(
+            "The source of the retention policy (example: Selectielijst gemeenten 2020)."
+        ),
+        max_length=255,
+        blank=False,
+        default="",
+    )
+    selectiecategorie = models.CharField(
+        _("selection category"),
+        help_text=_(
+            "The category as specified in the provided retention policy source (example: 20.1.2)."
+        ),
+        max_length=255,
+        blank=True,
+    )
+    archiefnominatie = models.CharField(
+        _("archive action"),
+        help_text=_("Determines if the archived data will be retained or destroyed."),
+        choices=ArchiveNominationChoices.choices,
+        max_length=50,
+        blank=False,
+        default=ArchiveNominationChoices.retain,
+    )
+    archiefactiedatum = models.DateField(
+        _("archive action date"),
+        help_text=_("Date when the publication will be archived or destroyed."),
+    )
```

Comment:
this must be nullable and not required, we can't calculate/determine this until we have the parameters from the related information categories.